Wangscape/Options.h

```diff
@@ -35,6 +35,7 @@ class Options
     std::vector<BorderModuleGroupLocation> leftBorderModuleGroups;
     std::vector<BorderModuleGroupLocation> topBorderModuleGroups;
     bool debugOutput;
+    std::pair<size_t, size_t> debugTileResolution;
```

Comment:
Oh, opened in December. I have to bring more code to Wangscape to get rid of all these issues.